app/src/organisms/RobotSettingsCalibration/CalibrationDataDownload.tsx

```diff
@@ -91,7 +91,12 @@ export function CalibrationDataDownload({
   }
 
   return (
-    <Flex justifyContent={JUSTIFY_SPACE_BETWEEN} alignItems={ALIGN_CENTER}>
+    <Flex
+      justifyContent={JUSTIFY_SPACE_BETWEEN}
+      alignItems={ALIGN_CENTER}
+      gridGap={SPACING.spacing40}
+      paddingBottom={SPACING.spacing24}
+    >
```

Comment:
```suggestion
    <Flex
      justifyContent={JUSTIFY_SPACE_BETWEEN}
      alignItems={ALIGN_CENTER}
      gridGap={SPACING.spacing40}
    >
```

with `paddingBottom={SPACING.spacing24}`
![Screenshot 2023-11-07 at 11 58 10 AM](https://github.com/Opentrons/opentrons/assets/474225/7a5166c7-465a-4a92-9a50-c7e98e0f17b2)


without `paddingBottom={SPACING.spacing24}`
![Screenshot 2023-11-07 at 12 00 59 PM](https://github.com/Opentrons/opentrons/assets/474225/1f4c13ee-3eb2-48ea-bcc0-568aa84af47b)



I think Brian fixed this spacing issue recently.